modules/travel_pay/app/services/travel_pay/expenses_client.rb

```diff
@@ -137,6 +137,39 @@ def delete_expense(veis_token, btsss_token, expense_id, expense_type)
       end
     end
 
+    ##
+    # Generic HTTP PATCH call to the BTSSS 'expenses' endpoints to update an expense
+    # Routes to appropriate endpoint based on expense type
+    #
+    # @param veis_token [String] VEIS authentication token
+    # @param btsss_token [String] BTSSS access token
+    # @param expense_id [String] UUID of the expense
+    # @param expense_type [String] Type of expense (EX: 'other')
+    # @param body [Hash] Request body to send to the API
+    # @return [Faraday::Response] API response
+    #
+    def update_expense(veis_token, btsss_token, expense_id, expense_type, body = {})
+      raise ArgumentError, 'Invalid expense_id' unless expense_id&.match?(TravelPay::Constants::UUID_REGEX)
+
+      endpoint_template = expense_endpoint_for_type(expense_type, :patch)
+      btsss_url = Settings.travel_pay.base_url
+      correlation_id = SecureRandom.uuid
+      endpoint = format(endpoint_template, expense_id: expense_id) # rubocop:disable Style/HashSyntax
```

Comment:
I think this is triggering on wanting the shorthand syntax. Don't disable the rule here, just do `format(endpoint_template, expense_id:)` if you can. I think this might be happening elsewhere in this file. If it's easy to fix in this PR, that would be good.